slstatus.c

```diff
@@ -759,7 +759,7 @@ int
 main(int argc, char *argv[])
 {
 	unsigned short int i;
-	char status_string[4096];
+	char status_string[2048];
```

Comment:
Why 2048 here? What if users (like me) want to use longer status bars? 😄 I don't think that 4096 is too much.